lib/puppet-syntax/tasks/puppet-syntax.rb

```diff
@@ -13,7 +13,11 @@ def filelist(paths)
     end
 
     def filelist_manifests
-      filelist("**/*.pp")
+      filelist("**/*.pp").exclude('plans/**/*')
```

Comment:
I went conservative on what we check for plans; this obviously won't work well in a control repo layout. However since we now have two things with the same extension that are parsed differently, the only way to identify them is with file path.

@hlindberg have a suggestion here? Should we just set `Puppet[:tasks] = true` for all manifests? Is there future work to make that possible?

Reply:
At this point differentiate by path is the best we can do. Not sure what you mean by always setting `Puppet[:tasks] = true`, obviously it should not be set for non plans / scripts...